test/e2e/config/ibmcloud-e2e.yaml

```diff
@@ -0,0 +1,58 @@
+managementClusterName: capi-ibmcloud-e2e
+
+providers:
+  - name: cluster-api
+    type: CoreProvider
+    versions:
+    - name: v1.0.2
+      value: https://github.com/kubernetes-sigs/cluster-api/releases/download/v1.0.2/core-components.yaml
+      type: url
+      files:
+      - sourcePath: "../data/shared/metadata.yaml"
+  - name: kubeadm
+    type: BootstrapProvider
+    versions:
+    - name: v1.0.2
+      value: https://github.com/kubernetes-sigs/cluster-api/releases/download/v1.0.2/bootstrap-components.yaml
+      type: url
+      files:
+      - sourcePath: "../data/shared/metadata.yaml"
+  - name: kubeadm
+    type: ControlPlaneProvider
+    versions:
+    - name: v1.0.2
+      value: https://github.com/kubernetes-sigs/cluster-api/releases/download/v1.0.2/control-plane-components.yaml
+      type: url
+      files:
+      - sourcePath: "../data/shared/metadata.yaml"
+  - name: ibmcloud
+    type: InfrastructureProvider
+    versions:
+    - name: v0.2.0
+      value: "${PWD}/config/default"
```

Comment:
where are we overriding the image name for the ibmcloud controller for the PR testing?

Reply:
It is currently using the default image(tagged main). Initially, I added the value as `infrastructure-components.yaml` from the latest version(0.2.0-alpha.1). Would that work or are you suggesting building an e2e image for the tests are overriding it to be the default image?